external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaOffsetReader.scala

```diff
@@ -88,10 +89,10 @@ private[kafka010] class KafkaOffsetReader(
     _consumer
   }
 
-  private val maxOffsetFetchAttempts =
+  private val fetchOffsetNumRetries =
```

Comment:
Is this renaming required for `Use CaseInsensitiveMap for KafkaOffsetReader`?

Reply:
In short, no. Similar detailed explanation applies just like in my previous comment.

Reply:
Well, the more I'm considering the renames inside `KafkaOffsetReader.scala` the more I feel it doesn't bring too much in a bugfix. It just ends-up in 2 lines change in the test so reverting them...

Reply:
Rolled back.